<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>来自老父亲的爱-口算题生成器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="cbAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>加法</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSub">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>减法</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMul">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>乘法</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbDiv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>除法</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>生成PDF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>5</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbMax">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>5</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbRate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>5</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAddMin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSubMin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMulMin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDivMin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDivMax">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSubMax">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>99</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAddMax">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>99</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMulMax">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDivRate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>220</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSubRate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>220</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAddRate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>220</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMulRate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>220</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMod">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>允许余数</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加减乘除混合运算</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
